Lib/fontbakery/commands/check_googlefonts.py

```diff
@@ -4,12 +4,16 @@
 
 from functools import partial
 from fontbakery.specifications.googlefonts import specification
-from fontbakery.commands.check_spec import main as super_main
+from fontbakery.commands.check_spec import (
+                                            runner_factory as super_runner_factory       
+                                          , main as super_main       
+                                          )
 
 # The values dict will probably get one or more specific blacklists
 # for the google font project. It would be good if it was not necessary
 # to copy paste this kind of configuration, thus a central init for
 # the google/fonts repository is be good.
 GOOGLEFONTS_SPECIFICS = {}
-
+runner_factory = partial(super_runner_factory, specification, values=GOOGLEFONTS_SPECIFICS)
```

Comment:
please add a comment here mentioning that Font Bakery Dashboard needs this!

Reply:
Alright, added.

Reply:
I'd like to resolve this in order to merge, but can't see a button for it :-/ I could however dismiss it.